package.json

```diff
@@ -114,6 +115,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.3",
     "dotenv": "^8.1.0",
+    "eslint": "^6.5.1",
```

Comment:
```suggestion
```

Reply:
I’ve removed it as I agree it’s not a part of this PR. I would suggest though adding all packages used in dev to project. I don’t have a global eslint and my IDE (VS code) throws an exception since it detects `.eslintrc` file. Just something to consider. I see that `babel-eslint` is referenced in package.json as well.